cypress/integration/recieve-notice/recieve-notice-after-checkout.spec.js

```diff
@@ -0,0 +1,147 @@
+import loanPolicy, { defaultLoanPolicy } from '../../support/fragments/circulation/loan-policy';
+import lostItemFeePolicy, { defaultLostItemFeePolicy } from '../../support/fragments/circulation/lost-item-fee-policy';
+import newPatronNoticePolicies from '../../support/fragments/circulation/newPatronNoticePolicies';
+import newPatronNoticeTemplate from '../../support/fragments/circulation/newPatronNoticeTemplate';
+import overdueFinePolicy, { defaultOverdueFinePolicy } from '../../support/fragments/circulation/overdue-fine-policy';
+import requestPolicy, { defaultRequestPolicy } from '../../support/fragments/circulation/request-policy';
+import materialTypesSettings, { defaultMaterialType } from '../../support/fragments/inventory/materialType/materialTypesSettings';
+import patronGroups from '../../support/fragments/settings/users/patronGroups';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import circulationRules from '../../support/fragments/circulation/circulation-rules';
+import settingsMenu from '../../support/fragments/settingsMenu';
+import checkoutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import TopMenu from '../../support/fragments/topMenu';
+
+describe('test name', () => {
+  const testPatronGroup = patronGroups.defaultPatronGroup;
+  const testPatronNoticeTemplate = newPatronNoticeTemplate.defaultUiPatronNoticeTemplate;
+  const testPatronNotice = newPatronNoticePolicies.defaultUiPatronNoticePolicies;
+  let ITEM_BARCODE;
+
+  beforeEach(() => {
+    ITEM_BARCODE = generateItemBarcode();
+
+    cy.getAdminToken()
+      .then(() => {
+        cy.getLoanTypes({ limit: 1 });
+        cy.getMaterialTypes({ limit: 1 });
+        cy.getLocations({ limit: 1 });
+        cy.getHoldingTypes({ limit: 1 });
+        cy.getHoldingSources({ limit: 1 });
+        cy.getInstanceTypes({ limit: 1 });
+        cy.getUsers({
+          limit: 1,
+          query: '"personal.firstName"="checkin-all" and "active"="true"',
+        });
+      })
+      .then(() => {
+        cy.getUserServicePoints(Cypress.env('users')[0].id);
+
+        cy.createInstance({
+          instance: {
+            instanceTypeId: Cypress.env('instanceTypes')[0].id,
```

Comment:
read value from method return value

Reply:
i'm doing it via **getInstanceTypes()**  on 50th line

Reply:
> 

i'm doing it through ** cy.getInstanceTypes({ limit: 1 });** on 50th line

Reply:
let's make a call about this comment